func_importvalue.go

```diff
@@ -0,0 +1,43 @@
+package cloudformation
+
+// ImportValue returns a new instance of ImportValue that imports valueToImport.
+func ImportValue(valueToImport string) ImportValueFunc {
```

Comment:
By my reading if "foo" were a parameter of the stack, `{"Fn::ImportValue": {"Ref": "foo"}}` it might be legal. 

Is there any way you'd be willing to do an experiment and find out for sure?